File: agent/library/src/main/java/org/matilda/logger/StderrLogger.java
package org.matilda.logger;

import java.util.Arrays;

public class StderrLogger implements Logger {

    @Override
    public void log(String message) {
        System.err.println(message);
    }

    @Override
    public void log(String message, Throwable throwable) {
        System.err.println(message);
        throwable.printStackTrace();
    }
}

File: agent/library/src/main/java/org/matilda/logger/StderrLogger.kt
package org.matilda.logger

class StderrLogger : Logger {
    override fun log(message: String) {
        System.err.println(message)
    }

    override fun log(message: String, throwable: Throwable) {
        System.err.println(message)
        throwable.printStackTrace()
    }
}
